js_modules/dagit/packages/core/src/typeexplorer/ConfigTypeSchema.tsx

```diff
@@ -88,16 +98,45 @@ function renderTypeRecursive(
         {type.isSelector && (
           <DictBlockComment indent={innerIndent} content="One of the following:" />
         )}
-        {type.fields.map((fieldData) => (
-          <DictEntry key={fieldData.name}>
-            <DictBlockComment indent={innerIndent} content={fieldData.description} />
-            {innerIndent}
-            <DictKey theme={props.theme}>{fieldData.name}</DictKey>
-            {!fieldData.isRequired && Optional}
-            {`: `}
-            {renderTypeRecursive(typeLookup[fieldData.configTypeKey], typeLookup, depth + 1, props)}
-          </DictEntry>
-        ))}
+        {type.fields.map((fieldData) => {
+          const keyDisplay = (
+            <DictKey
+              theme={props.theme}
+              style={
+                fieldData.defaultValueAsJson
+                  ? {borderBottom: `dashed ${Colors.Blue200} 1px`}
+                  : undefined
+              }
+            >
+              {fieldData.name}
+            </DictKey>
+          );
+          return (
+            <DictEntry key={fieldData.name}>
+              <DictBlockComment indent={innerIndent} content={fieldData.description} />
+              {innerIndent}
+              {fieldData.defaultValueAsJson ? (
+                <ValueTooltip
+                  content={
+                    <ConfigContent>{jsonString(fieldData.defaultValueAsJson)}</ConfigContent>
```

Comment:
While you're in here, could we check that this is a memoized react component at some level near here? I worry that if it's not, we might be introducing parsing of a lot of JSON into each render since these things are sort of unbounded in size. I don't /think/ this renders on every keystroke in the config editor, but there's a small chance it is...